packages/components/docs/token-table.tsx

```diff
@@ -1,6 +1,9 @@
 import * as React from "react";
 
-import { styled } from "twin.macro";
+import classNames from "classnames/bind";
+import styles from "./token-table.module.css";
+
+const cx = classNames.bind(styles);
```

Comment:
future thought -- might be nice to write a little util combining these 2 lines into 1 import

Reply:
Trying to think how that would be possible with the import syntax - we could write our own babel plugin. Is that what you mean?

Reply:
oh I didn't think it through very much... does sound like more effort than it's worth just to avoid a lil bit of repeated syntax